### PR TITLE
Freebox component with sensor and device tracker platforms

### PR DESCRIPTION
**Pull request by SNoof85**
**Description:**
This is the documentation for the new freebox component with sensor and device tracker platforms

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#18472

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by SNoof85** [comment]
I doubt this will be ready for the next release. The PR to implement the functions is still WIP because of an issue in the underlying aiofreepybox needed package to interact with the router.

File: source/_components/device_tracker.freebox.markdown [hunk 1]
```diff
@@ -1,55 +1,23 @@
 ---
 layout: page
-title: "Freebox"
-description: "Instructions on how to integrate Freebox routers into Home Assistant."
+title: "Freebox Device Tracker"
+description: "Instructions on how to track devices connected to a Freebox router into Home Assistant."
```
Comment (MartinHjelmare) [inline source/_components/device_tracker.freebox.markdown]: in Home Assistant

File: source/_components/device_tracker.freebox.markdown [hunk 2]
```diff
@@ -1,55 +1,23 @@
...
 date: 2018-05-16 23:00
 sidebar: true
 comments: false
 sharing: true
 footer: true
 logo: freebox.svg
 ha_category: Presence Detection
-ha_release: "0.70"
+ha_release: "0.85"
```
Comment (MartinHjelmare) [inline source/_components/device_tracker.freebox.markdown]: Don't change this. It's not a new platform.

File: source/_components/freebox.markdown [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+sharing: true
+footer: true
+logo: freebox.svg
+ha_category: Network
+ha_release: "0.85"
+ha_iot_class: "Local Polling"
+---
+
+
+The `freebox` component allow you to observe and control [Freebox router](http://www.free.fr/).
```
Comment (MartinHjelmare) [inline source/_components/freebox.markdown]: allows you to observe

File: source/_components/freebox.markdown [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+You must have set a password for your Freebox router web administration page and enabled the option "Permettre les nouvelles demandes d'associations".
+</p>
+
+The first time Home Assistant will connect to your Freebox, you will need to
+authorize it by pressing the right arrow on the facade of the Freebox when
+prompted to do so.
+
+### {% linkable_title Supported routers %}
+
+Only the routers with Freebox OS are supported :
```
Comment (MartinHjelmare) [inline source/_components/freebox.markdown]: Remove space before the colon.

**Comment by SNoof85** [comment]
All fixed I guess.

**Review by MartinHjelmare — APPROVED** [review]

File: source/_components/freebox.markdown [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+The first time Home Assistant will connect to your Freebox, you will need to
+authorize it by pressing the right arrow on the facade of the Freebox when
+prompted to do so.
+
+### {% linkable_title Supported routers %}
+
+Only the routers with Freebox OS are supported:
+* Freebox V6 also known as Freebox Revolution
+* Freebox mini 4k
```
Comment (pdecat) [inline source/_components/freebox.markdown:65]: Also works with Freebox v7 Delta router, thanks!

Comment (SNoof85) [reply source/_components/freebox.markdown:65]: Thx a lot for this information. I'll make a PR to add this.

**Pull request merged**